GameData/RealismOverhaul/RO_SuggestedMods/DodoLabs/Electron_RO.cfg

```diff
@@ -0,0 +1,651 @@
+//Sources:	https://space.skyrocket.de/doc_lau/electron.htm
+			https://www.rocketlabusa.com/space-systems/photon/
+			https://www.rocketlabusa.com/launch/electron/
+			https://www.rocketlabusa.com/updates/the-kick-stage-responsible-orbital-deployment/
+			https://en.wikipedia.org/wiki/Rocket_Lab_Electron
+
```

Comment:
These need to be commented out. Please do not leave plain text in .cfgs

Reply:
Oops, I overlooked that, thanks! It is fixed now.